terraform/aptos-node-testnet/main.tf

```diff
@@ -1,7 +1,3 @@
-terraform {
-  backend "s3" {}
```

Comment:
removing this since this is a reuse module and the backend config shouldn't be here (also terraform complained about this being here).